modules/data-loading/partials/kafka/kafka-data-source-details.adoc

```diff
@@ -13,61 +13,12 @@ To configure the data source object, the minimum requirement is the address of t
 .Data source configuration for external Kafka
 ----
 {
-"type": "mirrormaker",
-"source.cluster.bootstrap.servers": "<broker_addrs>"
+  "type": "mirrormaker",
+  "source.cluster.bootstrap.servers": "<broker_addrs>"
 }
 ----
 
-If the source cluster is configured for SSL or SASL protocols, you need to provide the following SSL/SASL credentials in order to communicate with the source cluster.
-
-* If the source cluster uses SASL, you need to upload the keytab of each Kerberos principal to every node of your TigerGraph cluster at the same absolute path.
 * If the source cluster uses SSL, see our documentation xref:tigergraph-server:data-loading:kafka-ssl-security-guide.adoc[]
```

Comment:
https://graphsql.atlassian.net/browse/DOC-1972?focusedCommentId=152682 see comment for `kafka-ssl-security-guide.adoc`.
* SSL for internal Kafka
* SSL for CRR
* SSL for loading from external Kafka

They should belong to 3 different categories(under 3 different pages) instead of putting them together.

Reply:
- Let's clarify here. We're not going to cover SSL for CRR here right?
- For internal and external Kafka, I don't see major difference. The "internal" Kafka is set up for testing loading from regular Kafka cluster including external Kafka. Let me know if there's any understanding gap. We can have a short discussion in Tuesday's meeting and get consensus about this with the team.  ( At the moment I still have a couple of JIRA tickets to handle, which also have high priorities).
- I am removing the SASL and the SASL/SSL parameters, because:  without SSL, SASL is not secured. SASL is authentication method, but without encryption, the connection is not secured. So we should simply recommend the users to enable regular SSL to secure the connection between Kafka clusters. User doesn't need to use both SASL + SSL. SSL is enough. And we didn't officially support SASL yet.

cc @arunramasami , FYI. Any comments regarding this DOC change I am making?